extensions/amp-analytics/amp-analytics.md

```diff
@@ -27,70 +27,72 @@ limitations under the License.
+Before you start using AMP analytics on your site, you need to decide whether you will you use third-party analytics tools to analyze user engagement, or your own in-house solution.
+
+{% call callout('Read on', type='read') %}
+Learn all about AMP analytics in the [Configure Analytics](https://www.ampproject.org/docs/guides/analytics_amp) guide.
+{% endcall %}
+
+### Sending data to an analytics vendor <a name="analytics-vendors"></a>
+
+AMP analytics is specifically designed to measure once and report to many. If you are already working with one or more analytics vendors, check the list of [Analytics Vendors](https://www.ampproject.org/docs/guides/analytics/analytics-vendors.html) to see if they’ve integrated their solution with AMP.
+
+
+To send data to a supported analytics vendor:
+
+1.  In the `<amp-analytics>` tag, add the `type`attribute and set its value to the specified [vendor](https://www.ampproject.org/docs/guides/analytics/analytics-vendors.html).
+2. Determine what data you want to capture and track, and specify those details in the configuration data. See the vendor's documentation for  instructions on how to capture analytics data.
 
-**Example**
 
-In the following example, we send analytics data to `https://example.com/analytics` when the AMP document is first loaded, and each time an `<a>` tag is clicked:
+*Example: Sending data to a third-party analytics provider*
+
+In the following example, analytics data is sent to Nielsen, a third-party analytics provider. Details for configuring analytics data for Nielsen can be found in the [Nielsen](https://engineeringportal.nielsen.com/docs/DCR_Static_Google_AMP_Cloud_API) documentation.
 
 ```html
-<amp-analytics>
-<script type="application/json">
-{
-  "requests": {
-    "pageview": "https://example.com/analytics?url=${canonicalUrl}&title=${title}&acct=${account}",
-    "event": "https://example.com/analytics?eid=${eventId}&elab=${eventLabel}&acct=${account}"
-  },
-  "vars": {
-    "account": "ABC123"
-  },
-  "triggers": {
-    "trackPageview": {
-      "on": "visible",
-      "request": "pageview"
-    },
-    "trackAnchorClicks": {
-      "on": "click",
-      "selector": "a",
-      "request": "event",
+<amp-analytics type="nielsen">
+    <script type="application/json">
+    {
       "vars": {
-        "eventId": "42",
-        "eventLabel": "clicked on a link"
+        "apid": "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX",
+        "apv": "1.0",
+        "apn": "My AMP Website",
+        "section": "Entertainment",
+        "segA": "Music",
+        "segB": "News",
+        "segC": "Google AMP"
       }
     }
-  }
-}
-</script>
+    </script>
 </amp-analytics>
 ```
 
-## Attributes
+### Sending data in-house
```

Comment:
A couple considerations: This could also be used for a non-directly supported vendor, not only strictly in-house. Do you have an idea how to handle it?